packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/picture-element.json

```diff
@@ -1,18 +1,18 @@
 {
   "picture": {
-    "defaultImage": "../../assets/images/placeholder/400x300.png",
+    "defaultImage": "../../assets/images/placeholder/1600x600.png",
     "srcSets": [
       {
-        "url": "../../assets/images/placeholder/900x700.png",
-        "width": "1050"
+        "url": "../../assets/images/placeholder/1600x600.png",
```

Comment:
I tested the picture component and it doesn't work --> https://mayflower.digital.mass.gov/patternlab/b/DP-22263_implement-responsive-images/?p=atoms-picture-element

I think the order matters since now it's setting max-width instead of min-width, the media sizes should go from the smallest value to the biggest value. Otherwise, the first condition will be met and will not use a smaller image. 

The data need to be passed in in the order of 480, 910, 1200. Is there a way to better enforce this in code? If not, we should also add a note in mayflower and openmass about this.

Reply:
Hi @clairesunstudio , I reversed the order of these, seems to be working well now, let me know any other detail.